src/models/user.js

```diff
@@ -54,8 +54,12 @@ const userSchema = new mongoose.Schema(
     },
     password: {
       type: String,
+      select: false,
+    },
+    googleId: {
+      type: String,
+      select: false,
     },
```

Comment:
`select: false` is good for queries where we want these fields to be hidden.

But what about queries where we need password? Like auth query for password-based login.
I think you'll have to add `.select('password')` at the end of those queries.